packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1181,6 +1181,117 @@ class RenderClipPath extends _RenderCustomClip<Path> {
   }
 }
 
+/// The shape of a physical model.
+enum PhysicalModelShape {
+  /// Rounded corner rectangle.
+  rrect,
+
+  /// Oval.
```

Comment:
Technically we're not supposed to write dartdocs that someone could write simply by reading the name of the thing you're documenting.  For example, see how a similar value is documented for BoxShape:

https://docs.flutter.io/flutter/painting/BoxShape-class.html

Actually, can we just use BoxShape instead of PhysicalModelShape ?